e2e/tests/nightly/owner-copy-to-workspace.spec.ts

```diff
@@ -27,7 +27,7 @@ let defaultCdrWorkspace: string;
  * @param {string} sourceWorkspaceName: Source workspace name
  * @param {string} to create new destination workspace with CDR Version
  */
-describe('Workspace owner Jupyter notebook action tests', () => {
```

Comment:
Probably worth having a comment somewhere in this file which explains rationale for  making this a nightly test rather than an normal - it's not clear without deep inspection of the test case

Reply:
Comment added. Thank you.